### PR TITLE
Allow copying of archive items to display sets

### PR DESCRIPTION
**Pull request by MikeOverkamp-diag**
Note: I intentionally did not include a link to the new view anywhere yet, as it is only intended for beta testing. I will manually supply the correct url to the testers.

### CODE REVIEW

**Review by jmsmkn — COMMENTED** [review]
Looks good but could you add a test for the reader studies that are selectable and for the overall view?

**Comment by jmsmkn** [comment]
Maybe I'm missing something I don't see how that test tests for reader study selection and the overall view. Maybe I wasn't clear. I want to see a test to ensure that:

```
        self.fields["reader_study"].queryset = (
            get_objects_for_user(
                self.user,
                "reader_studies.change_readerstudy",
                accept_global_perms=False,
            )
            .filter(use_display_sets=True)
            .order_by("title")
        )
```

and 

```
    permission_required = (
        f"{Archive._meta.app_label}.use_{Archive._meta.model_name}"
    )
```

are implemented.

File: app/tests/archives_tests/test_views.py [hunk 1]
```diff
@@ -456,3 +457,67 @@ def test_api_archive_item_add_and_update_non_image_file(client, settings):
...
+        viewname="archives:items-reader-study-update",
+        client=client,
+        reverse_kwargs={"slug": archive.slug},
+        follow=True,
+        user=editor,
+    )
+
+    assert response.status_code == 200
+    assert str(rs1.pk) in response.rendered_content
+    assert str(rs2.pk) not in response.rendered_content
```
Comment (MikeOverkamp-diag) [inline app/tests/archives_tests/test_views.py:502]: The queryset test is included by these two asserts. I'll add a test for someone without edit rights.

**Comment by MikeOverkamp-diag** [comment]
Now that I'm working on the test, I'm unsure if the permission I used is the one we want. Do we want all archive users to be able to add items to a reader study? I know that's the current implementation for images as well, but if we want to chance this, I'd say this is the time.

**Comment by MikeOverkamp-diag** [comment]
Nevermind my previous comment, I don't see why we wouldn't want user to be able to do this. 

I think I covered all possible scenarios in the tests now.

**Comment by jmsmkn** [comment]
> Nevermind my previous comment, I don't see why we wouldn't want user to be able to do this.
> 
> I think I covered all possible scenarios in the tests now.

Yes I think it's right, otherwise you end up with users downloading the images from the archive and re-uploading them to the study.

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**